### PR TITLE
Chapter 4: Rails-flavored Ruby

### PR DESCRIPTION
**Pull request by vannam1996**

### CODE REVIEW

**Comment by vannam1996** [comment]
image

**Comment by vannam1996** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
```
Comment (dothidiemthao) [inline app/helpers/application_helper.rb]: i18n

**Comment by dothidiemthao** [comment]
commented

**Comment by vannam1996** [comment]
ready

**Review by dothidiemthao — APPROVED** [review]

**Comment by dothidiemthao** [comment]
merged

**Pull request merged**